src/nft/components/bag/profile/ListingButton.tsx

```diff
@@ -90,7 +113,7 @@ export const ListingButton = ({ onClick, buttonText, showWarningOverride = false
       listingsAboveSellOrderFloor,
       invalidPrices,
     ]
-  }, [sellAssets])
+  }, [isNftListV2, sellAssets, showResolveIssues, toggleShowResolveIssues])
```

Comment:
tech debt: I almost think this memo should be made into a hook its gotten rather long

Reply:
also looking at the memo below - considering most of its inputs are whats being returned here that could also be turned into a hook and could call this new hook - just some thoughts to cleanup the file a little

Reply:
Discussed offline, will be refactoring ListButton styles and logic post merge to make simplification easier